rosie/core/classifiers/invalid_cnpj_cpf_classifier.py

```diff
@@ -20,17 +19,16 @@ class InvalidCnpjCpfClassifier(TransformerMixin):
     recipient_id : string column
         A CNPJ (Brazilian company ID) or CPF (Brazilian personal tax ID).
     """
-
-    def fit(self, X):
+    def fit(self, dataframe):
         return self
 
-    def transform(self, X=None):
+    def transform(self, dataframe=None):
         return self
 
-    def predict(self, X):
-        return np.r_[X.apply(self.__is_invalid, axis=1)]
-
-    def __is_invalid(self, row):
-        document_types = ['bill_of_sale', 'simple_receipt', 'unknown']
-        return (row['document_type'] in document_types) \
-            & (not cpfcnpj.validate(str(row['recipient_id'])))
+    def predict(self, dataframe):
+        def is_invalid(row):
```

Comment:
Why do we have a method inside another method here? Couldn't we have a auxiliary method in this case?

Reply:
We have a function inside a method because in my opinion it doesn't make sense having it in the class space: It's not being reused anywhere, it makes no use of `self`. An auxiliary method is something which abstracts stateful logic within the object itself, this is not a method at all.

It's just a dummy function whose only purpose is being passed to `apply`, thats why I argue it should be kept in the most local scope possible, that reinforces the idea that it's not a core part of the class but rather something that is disposable.

Me and @cuducos had a thread that sorta touched on this subject, [you can read it here](https://github.com/datasciencebr/rosie/pull/95/files/a30feb8e94c744128c2051916ee8841edf2b4e49).